python/lsst/afw/cameraGeom/cameraConfig.py

```diff
@@ -61,7 +61,7 @@ class CameraConfig(pexConfig.Config):
     # scaled by the plate scale in appropriate units
     radialCoeffs = pexConfig.ListField(
         "Coefficients for radial distortion", float)
-    # The following is commented until radialXYTransform supports an offset (ticket/3155)
+    # The following is commented until radialTransform supports an offset (ticket/3155)
```

Comment:
This looks frighteningly like a Trac ticket number.  Given that we haven't needed this so far, perhaps we should just delete the commented-out code?  The alternative would be creating a JIRA ticket, but I bet it'd either linger or get marked Won't Fix.